mern/client/src/screens/StartingIntent/StartingIntent.js

```diff
@@ -1,94 +1,122 @@
-import React from "react";
```

Comment:
This file seems very large. The code isn't super readable and we should probably separate out some functions here.